src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITupleExpression.vb

```diff
@@ -0,0 +1,330 @@
+        Public Sub TupleExpression_UserDefinedConversionToTupleType()
+            Dim source = <![CDATA[
+Imports System
+
+Class C
+    Private ReadOnly _x As Integer
+    Public Sub New(x As Integer)
+        _x = x
+    End Sub
+
+    Public Shared Widening Operator CType(x As (Integer, String)) As C
+        Return New C(x.Item1)
+    End Operator
+
+    Public Shared Widening Operator CType(c As C) As (Integer, String)
+        Return (c._x, c._x.ToString)
+    End Operator
+
+    Public Sub M(c1 As C)
+        Dim t As (Integer, String) = c1'BIND:"Dim t As (Integer, String) = c1"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Dim t As (I ... tring) = c1')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 't')
+    Variables: Local_1: t As (System.Int32, System.String)
+    Initializer: IConversionExpression (ConversionKind.Basic, Implicit) (OperationKind.ConversionExpression, Type: (System.Int32, System.String)) (Syntax: 'c1')
+        IConversionExpression (ConversionKind.OperatorMethod, Implicit) (OperatorMethod: Function C.op_Implicit(c As C) As (System.Int32, System.String)) (OperationKind.ConversionExpression, Type: C) (Syntax: 'c1')
+          IParameterReferenceExpression: c1 (OperationKind.ParameterReferenceExpression, Type: C) (Syntax: 'c1')
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
+
+        <Fact, WorkItem(10856, "https://github.com/dotnet/roslyn/issues/10856")>
+        Public Sub TupleExpression_InvalidConversion()
+            Dim source = <![CDATA[
+Class C
+    Private ReadOnly _x As Integer
+    Public Sub New(x As Integer)
+        _x = x
+    End Sub
+
+    Public Shared Widening Operator CType(value As Integer) As C
+        Return New C(value)
+    End Operator
+
+    Public Shared Widening Operator CType(c As C) As Integer
+        Return CShort(c._x)
+    End Operator
+
+    Public Shared Widening Operator CType(c As C) As String
+        Return c._x.ToString()
+    End Operator
+
+    Public Sub M(c1 As C)
+        Dim t As (Short, String) = (New C(0), c1)'BIND:"Dim t As (Short, String) = (New C(0), c1)"
+    End Sub
+End Class
+
+]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement, IsInvalid) (Syntax: 'Dim t As (S ... w C(0), c1)')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration, IsInvalid) (Syntax: 't')
+    Variables: Local_1: t As (System.Int16, System.String)
+    Initializer: ITupleExpression (OperationKind.TupleExpression, Type: (System.Int16, c1 As System.String), IsInvalid) (Syntax: '(New C(0), c1)')
```

Comment:
>ITupleExpression (OperationKind.TupleExpression, Type: (System.Int16, c1 As System.String), IsInvalid) (Syntax: '(New C(0), c1)') [](start = 17, length = 129)

Given other tests, it is not clear if we want to have a conversion on top of this node.